app/__init__.py

```diff
@@ -25,3 +27,35 @@ def resume():
 @app.route('/contact')
 def contact(): 
     return render_template('contact.html', title="Contact", url=os.getenv("URL"))
+
+@app.route('/send-email', methods=['GET','POST'])
+def send_email():
+    # HTTP POST Request args
+    email_sender = request.form['email']
+    name = request.form['name']
+    subject = request.form['subject']
+    message_content = request.form['message']
+
+    print(email_sender, name, subject, message_content)
+
+    # Data from env
+    email_server = os.environ.get('MAIL_SERVER')
+    email_server_port = os.environ.get('MAIL_SMPT_PORT')
+    email_username = os.environ.get('MAIL_USERNAME')
+    email_password = os.environ.get('MAIL_PASSWORD')
+    email_recipent = os.environ.get('MAIL_RECIPENT')
+
+    # Email Data
+    msg = MIMEText("Name:"+name+"\nContact email: "+email_sender+"\nMessage: "+message_content)
+    msg['Subject'] = subject
+    msg['From'] = email_username
+    msg['To'] = email_recipent
+
+    
+
+    server = smtplib.SMTP_SSL(email_server, email_server_port)
+    server.login(email_username, email_password)
+    server.sendmail(email_username, [email_recipent], msg.as_string())
+    server.quit()
+    return render_template('index.html', title="MLH Fellow", response="Your message was sent succesfully", url=os.getenv("URL"))
```

Comment:
I think it would make more sense to the user if we reload the contact page with a banner above the contact form containing the message, rather than switching to the index page